scvdb.yml

```diff
@@ -5,23 +5,135 @@ more-info: 'https://consensys.github.io/smart-contract-best-practices/ESCVDB'
 
 vulnerabilities:
   - id: 1
-    name: 'Example Vulnerability Entry'
-    classification: 'EVM' # I think the might be 'EVM' or 'Common logical error' or 'Front Running'?
-    description: > # `>` allows for a multiline entry
-      This is an example of the types of data that would be included about a given vulnerability.
-      The issue can is caused by...
-      The issue can result in...
-      The issue can be mitigated by...
-    tags: [underflow, arithmetic]
-    more-info: 
-      - `https://any_place_with_more_info.com`
-      - `https://consensys.github.io/smart-contract-best-practices/{something we wrote about it}`
+    name: 'Reentrancy'
+    classification: 'Race Condition'
+    description: >
+      Occurs when a contract makes a call to an external contract, which calls
```

Comment:
Please shorten the descriptions to be much more succinct. For example, in the issue I wrote this one as: 

> description: 'Occurs when a contract makes a call to an external contract, 
> which calls back into the calling contract, making an undesired state change.'

My thinking is that there will always be more information required about these issues, so it's best to provide a short description, and then links to other resources. BTW, I really like the "more info" links you provided for a lot of these vulns.

Reply:
Sure, sounds good to me. I will get some new descriptions in there. Thanks!